Content.Server/Radio/EntitySystems/JammerSystem.cs

```diff
@@ -73,17 +103,24 @@ private void OnActivate(EntityUid uid, RadioJammerComponent comp, ActivateInWorl
     private void OnPowerCellChanged(EntityUid uid, ActiveRadioJammerComponent comp, PowerCellChangedEvent args)
     {
         if (args.Ejected)
-            RemComp<ActiveRadioJammerComponent>(uid);
+        {
+            ChangeLEDState(false, uid, EntityManager.GetComponentOrNull<RadioJammerComponent>(uid));
```

Comment:
Acquiring the component like this should not be necessary, you can make the ChangeLEDState function resolve the component. I'll go into detail in this. Alternatively you could use TryComp and handle it accordingly.